src/screens/SearchBeers/styles.scss

```diff
@@ -102,3 +102,16 @@
   border: 2px solid $myGray;
   outline: 0;
 }
+
+.button-clean-filter {
+  width: 130px;
+  height: 30px;
+  border: 2px solid $myGray;
+  opacity: 0.6;
+  outline: 0;
+  background-color: $mySmokewhite;
+}
+
+.button-clean-filter:hover {
```

Comment:
Use sass here

Reply:
You should also keep pattern of buttons that are already in the app. All buttons have border-radius and background - this one does not.